<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字体大小</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fontBox">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>60</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="value" stdset="0">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>灰度级别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>伽马校正</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="gammaBox">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>140</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>9.900000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>2.200000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>setting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setting()</slot>
  <slot>preview()</slot>
 </slots>
</ui>
